README-ta.md

```diff
@@ -0,0 +1,82 @@
+
+## `README.md` உள்ளூர்மயமாக்கல்
+|  |  |
+|---|---|
+|[French README](README-fr.md)|[Korean README](README-ko.md)|
+|[German README](README-de.md)|[Portuguese README](README-pt.md)|
+|[Hindi README](README-hi.md)|[Spanish README](README-es.md)|
+|[Indonesian README](README-id.md)|[Chinese README](README-zh.md)|
+|[Japanese README](README-ja.md)|[Tamil README](README-ta.md)|
+|[Vietnamese README](README-vi.md)|[Russian README](README-ru.md)|
+|[Italian README](README-it.md)|[Polish README](README-pl.md)|
+
+## டாக்கர் பயன்படுத்தி உள்-வலைத்தளத்தை இயக்குவதற்கான வழி
+
+குபெர்னெட்ஸ் வலைத்தளத்தை உள்ளே இயக்குவதற்கான பரிந்துரைக்கப்பட்ட வழி, [ஹ்யூகோ] (https://gohugo.io) என்ற 'ஸ்டாடிக்' வலைத்தள ஜெனரேட்டரை உள்ளடக்கிய ஒரு சிறப்பு [டாக்கர்] (https://docker.com) இமேஜ்-ஐ இயக்க வேண்டும்.
+
+> நீங்கள் விண்டோஸில் இயங்குகிறீர்கள் என்றால், நீங்கள் நிறுவக்கூடிய இன்னும் சில கருவிகள் உங்களுக்குத் [Chocolatey](https://chocolatey.org). `choco install make` தேவைப்படும்.
+
+> நீங்கள் வலைத்தளத்தை டாக்கர் இல்லாமல் உள்-வலைத்தளத்தை இயக்க விரும்பினால், இதை பாருங்கள் [ஹ்யூகோவைப் பயன்படுத்தி உள்-வலைத்தளத்த இயக்குவது](#ஹ்யூகோவைப்-பயன்படுத்தி-உள்-வலைத்தளத்த-இயக்கும்-முறை) கீழே.
+
+உங்களிடம்  [இயங்கும் டாக்கர் இருந்தால்](https://www.docker.com/get-started), `kubernetes-hugo` என்ற இமேஜை டாக்கர் பயன்படுத்தி உள்-வலைத்தளத்தில் இயக்கலாம்:
+
+```bash
+make docker-image
+```
+
+இமேஜ் கட்டப்பட்டதும், நீங்கள் உள்-வலைத்தளத்தை இயக்கலாம்:
+
+```bash
+make docker-serve
+```
+
+வலைத்தளத்தைக் காண உங்கள் உலாவியை http://localhost:1313 க்குத் திறக்கவும். மூல கோப்புகளில் நீங்கள் மாற்றங்களைச் செய்யும்போது, ஹ்யூகோ வலைத்தளத்தைப் புதுப்பித்து உலாவி புதுப்பிக்கப்படும்.
+
+## ஹ்யூகோவைப் பயன்படுத்தி உள்-வலைத்தளத்த இயக்கும் முறை
+
+ஹ்யூகோ நிறுவல் வழிமுறைகளுக்கு [அதிகாரப்பூர்வ ஹ்யூகோ ஆவணங்கள்] (https://gohugo.io/getting-started/installing/) ஐப் பார்க்கவும். [`Netlify.toml`](netlify.toml#L9) கோப்பில்` HUGO_VERSION` என்விரோன்மெண்ட் மாறியால் குறிப்பிடப்பட்ட ஹ்யூகோ நீட்டிக்கப்பட்ட பதிப்பை நிறுவுவதை உறுதிசெய்க.
+
+நீங்கள் ஹ்யூகோவை நிறுவியதும் உள்-வலைத்தளத்தை இயக்க:
+
+```bash
+make serve
+```
+
+வலைத்தளத்தைக் காண உங்கள் உலாவியை http://localhost:1313 க்குத் திறக்கவும். மூல கோப்புகளில் நீங்கள் மாற்றங்களைச் செய்யும்போது, ஹ்யூகோ வலைத்தளத்தைப் புதுப்பித்து உலாவி புதுப்பிக்கப்படும்.
+
+## சமூகம், கலந்துரையாடல், பங்களிப்பு மற்றும் ஆதரவு
+
+[சமூகப் பக்கத்தில்](http://kubernetes.io/community/) குபெர்னெட்ஸ் சமூகத்துடன் எவ்வாறு ஈடுபடுவது என்பதை அறிக.
+
+இந்த திட்டத்தின் பராமரிப்பாளர்களை நீங்கள் இங்கு அணுகலாம்:
+
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+
+### நடத்தை விதி
+
+குபெர்னெட்ஸ் சமூகத்தில் பங்கேற்பது [குபெர்னெட்ஸ் நடத்தை விதிமுறை](code-of-conduct.md) ஆல் நிர்வகிக்கப்படுகிறது.
+
+## நன்றி!
+
+குபெர்னெட்ஸ் சமூக பங்களிப்பை வளர்த்துக் கொள்கிறது, மேலும் எங்கள் வலைத்தளத்திற்கும் எங்கள் ஆவணங்களுக்கும் நீங்கள் செய்த பங்களிப்புகளை நாங்கள் பாராட்டுகிறோம்!
```

Comment:
nit : Change to குபேர்ண்ட்ஸ் சமூக பங்களினால் வளர்ந்து கொண்டிருக்கிறது instead of குபெர்னெட்ஸ் சமூக பங்களிப்பை வளர்த்துக் கொள்கிறது

Reply:
Done :+1: